chart/examples/minikube/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   # image.repository -- The name of the docker repository for fusionauth-app
   repository: fusionauth/fusionauth-app
   # image.repository -- The docker tag to pull for fusionauth-app
-  tag: 1.30.1
+  tag: 1.32.1
```

Comment:
This tag doesn't seem to exist yet in docker hub.

Reply:
It doesn't yet.. Waiting for @robotdan to publish it. :)